m2cgen/interpreters/python/interpreter.py

```diff
@@ -9,9 +9,11 @@ class PythonInterpreter(ToCodeInterpreter,
     # 60 raises MemoryError for some SVM models with RBF kernel.
     bin_depth_threshold = 55
 
-    exponent_function_name = "np.exp"
-    power_function_name = "np.power"
-    tanh_function_name = "np.tanh"
+    exponent_function_name = "math.exp"
+    power_function_name = "math.pow"
+    tanh_function_name = "math.tanh"
+
+    with_vector_operations = False
```

Comment:
Not sure why can't we keep using `with_vectors` flag here that comes from `ToCodeInterpreter`?

Reply:
`with_vectors` means that some vectors are used. Refer to https://github.com/BayesWitnesses/m2cgen/blob/master/generated_code_examples/python/classification/linear.py as an example.
`with_vector_operations` means that some vector math happens. Refer to https://github.com/BayesWitnesses/m2cgen/blob/30abeef339012c9e5ab9132513ebb9d6d6b7d0b4/generated_code_examples/python/classification/random_forest.py#L29 as an example.

Reply:
@izeigerman Oh, just realized that `with_vector_operations` is the same as `with_linaer_algebra`.
Updated in the latest commit.